src/style.css

```diff
@@ -2,0 +1,62 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.form__input {
+  box-sizing: border-box;
+  display: flex;
+  width: 100%;
```

Comment:
Don't fix container size (if there is no such a requirement). You've set the width to 100% here, which is a good practice as it allows the content to dictate the size of the container.